src/win/fs-event.c

```diff
@@ -159,14 +159,20 @@ int uv_fs_event_start(uv_fs_event_t* handle,
   uv__handle_start(handle);
 
   /* Convert name to UTF16. */
-  name_size = uv_utf8_to_utf16(path, NULL, 0) * sizeof(WCHAR);
+
+  name_size = MultiByteToWideChar(CP_UTF8, 0, path, -1, NULL, 0) *
+              sizeof(WCHAR);
```

Comment:
if all arguments don't fit in one line, we tend to align them on one line each (yeah yeah, src/win/ is not consistent...) :-)